Backpack/src/androidTest/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarControllerTest.kt

```diff
@@ -49,7 +49,7 @@ class BpkCalendarControllerTest {
 
   @Before
   fun setUp() {
-    AndroidThreeTen.init(InstrumentationRegistry.getInstrumentation().targetContext)
+    AndroidThreeTen.init(TestContext)
```

Comment:
created a property that also includes setting the theme which fixed a few issues for other tests